ccdproc/bitmask.py

```diff
@@ -0,0 +1,473 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import collections
+import itertools
+
+import numpy as np
+
+_INTEGER_DTYPE_KINDS = {'u', 'i'}
+
+# Powers of two 2**0 - 2**63
+_POWERS = (
```

Comment:
Man, I've never seen anything like this! Please, please, get rid of ``_POWERS``!